Make decoration note consistent with other I18N specs

aphillips opened the pull request:
I have been making all of our current WG Note documents use a consistent "example req" note. In a recent PR, @r12a commented out the `gap` style that is not currently used. In this PR I restore it using the correct style name (`issue`) and add the example of a terminology definition.

Note that we use the style `issue-example` (which I add to `local.css`) because `issue` has an auto-counter we don't want to trigger.

An argument can be made that we should omit the `issue` example until it is needed by this document. Thoughts?


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/aphillips/string-meta/pull/71.html" title="Last updated on Jul 28, 2022, 3:46 PM UTC (01671f7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/string-meta/71/905a6b2...aphillips:01671f7.html" title="Last updated on Jul 28, 2022, 3:46 PM UTC (01671f7)">Diff</a>

aphillips committed:
Fix formatting issues and contact info

Update the styles to match other I18N WG notes and fix @aphillips's contact info

- Remove the `gap` style and replace with regular `issue` style
- Add `issue-example` style (to avoid the auto-counter in the `issue` style) for the note about styles
- Restyle content in example 1 and make it an `aside` so that it auto-counts

aphillips committed:
Merge branch 'w3c:gh-pages' into gh-pages

aphillips committed:
Removed unused styles from local.css

The `gap` style is no longer used.

aphillips committed:
Merge branch 'gh-pages' of https://github.com/aphillips/string-meta into gh-pages

aphillips committed:
Merge branch 'gh-pages' of https://github.com/aphillips/string-meta into gh-pages

aphillips committed:
Make decoration note consistent with other I18N specs

I have been making our other notes consistent in their use of decoration and styling. Although this document doesn't currently have any issues, I restored that example in case we add any in the future. I also added the "definition" example.

r12a commented:
I was looking at this yesterday and it seems to me that the content of this note is in the wrong place.  I think we should:
1. remove the note markup
2. put it in it's own subsection called something like "Conventions"
3. move that section to the Introduction, immediately before the Terminology subsection (so that terminology conventions are described before they are used)

I also think that we should at least comment out (not remove) the gap description until we have some gaps.  However, that said, i think it would actually make more sense to the reader to use _issue_ markup for these items.  I also like that issues have a heading that says what it is.

aphillips commented:
This particular document has the note in a weird place. Our other docs have it in different places, but it generally goes were the 2119 keyword definition goes.

We could put it into a potted section we include like "Conventions", although I note that many of our docs have "Terminology and Notation" types of section.

This PR changes to use `issue` notation for gaps--not `gap`.

In the editor's copy I favor leaving uncommented. We can comment it out when publishing to TR if there are no open issues.

r12a commented:
> In the editor's copy I favor leaving uncommented. We can comment it out when publishing to TR if there are no open issues.

Despite echidna's promise, it's still often a pain to publish documents, and if i'm publishing i'd rather not have to remember to comment that out each time.  Why don't you want to comment it out if there are no such issues?

aphillips commented:
I don't want to have to remember to put it back as a style when I use `issue`! Actually, we could automate `issue-example`'s appearance with a little JS at the end of the document (checking on the value of the `issue` counter and only showing the style if non-zero). I'll try adding that to the PR.

aphillips committed:
move and fix document convention block [I18N-ACTION-1177]

Per discussion in 2022-07-28 telecon, move the document conventions to their own section of the introduction and delete redundant bullet list.

aphillips commented:
Merging so that I can complete my action item. Can fix any styling nits as needed.
